### PR TITLE
Feature/demo2 last changes

### PR DESCRIPTION
**Pull request by gitrepository100**
Last changes to Demo-2 commited

### CODE REVIEW

File: OneSchedule/Startup.cs [hunk 1]
```diff
@@ -35,13 +37,57 @@ public void ConfigureServices(IServiceCollection services)
...
+            // if you are using persistent job store, you might want to alter some options
+            services.Configure<QuartzOptions>(options =>
+            {
+                options.Scheduling.IgnoreDuplicates = true; // default: false
+                options.Scheduling.OverWriteExistingData = true; // default: true
+            });
+
+            services.AddQuartz(q =>
+            {
+                // handy when part of cluster or you want to otherwise identify multiple schedulers
```
Comment (elkavitaly) [inline OneSchedule/Startup.cs]: remove comments

File: OneSchedule.Updater/appsettings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ﻿{
-  "RequstUri": "https://localhost:5001/api/update"
+  "RequstUri": "http://localhost:5000/api/update"
```
Comment (elkavitaly) [inline OneSchedule.Updater/appsettings.json]: Don't push urls to service

File: OneSchedule.Domain/NotificationScheduler.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+using Quartz.Impl;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OneSchedule.Domain
+{
+    public class NotificationScheduler
```
Comment (elkavitaly) [inline OneSchedule.Domain/NotificationScheduler.cs]: Do you need this class?

File: OneSchedule.Domain.Abstractions/INotificationSender.cs [hunk 4]
```diff
@@ -4,6 +4,6 @@ namespace OneSchedule.Domain.Abstractions
 {
     public interface INotificationSender
     {
-        public Task SendNotification();
+        public async Task SendNotification() { }
```
Comment (elkavitaly) [inline OneSchedule.Domain.Abstractions/INotificationSender.cs]: Why this was changed?

**Review by elkavitaly — CHANGES_REQUESTED** [review]

**Review by elkavitaly — APPROVED** [review]

**Review by VallarDark — APPROVED** [review]

**Pull request merged**